app/controllers/admin/administrators/administrators_controller.rb

```diff
@@ -47,6 +47,9 @@ def create
         ActiveRecord::Base.transaction do
           user.save!
           AdminProfile.create!(user: user)
+          AdminMailer.account_created_email(
+            user, helpers.new_user_session_url
```

Comment:
So the welcome email would have a link to the sign in page. However, personally I think it would be more convenient for the user if this were a magic link that they could click and log in. Just wondering what others think.

Reply:
It should be a link to the sign in page, because chances are the magic link will have expired by the time they get to the email. The user isn't the one who triggers this

Reply:
That's what Al specified anyway